orchagent/portsorch.cpp

```diff
@@ -871,12 +879,12 @@ bool PortsOrch::addVlanMember(Port vlan, Port port)
     attr.value.u16 = vlan.m_vlan_id;
     attrs.push_back(attr);
 
-    attr.id = SAI_VLAN_MEMBER_ATTR_PORT_ID;
+    attr.id = SAI_VLAN_MEMBER_ATTR_BRIDGE_PORT_ID;
     attr.value.oid = port.m_port_id;
```

Comment:
We will not consider LAG under bridge scenario in SONiC.

---
In reply to: [112708538](https://github.com/lguohan/sonic-swss/pull/1#discussion_r112708538) [](ancestors = 112708538)